src/controller/c/robot_private.h

```diff
@@ -48,6 +48,8 @@ const char *robot_get_device_name(WbDeviceTag tag);
 const char *robot_get_device_model(WbDeviceTag tag);
 void robot_mutex_lock_step();
 void robot_mutex_unlock_step();
+int robot_step_begin(int duration);
+int robot_step_end();
```

Comment:
This is not needed as these functions are used only in the robot module and should be static:
```suggestion
```